bears/jinja2/Jinja2Bear.py

```diff
@@ -184,6 +189,45 @@ def check_for_variable_spacing_issues(self,
                     end_column=m.end(0) + 1,
                     diffs=diff)
 
+    def check_for_statement_spacing_issues(self,
+                                           file,
+                                           filename,
+                                           line,
+                                           line_number,
+                                           statement_spacing):
+        """
+        Checks any statement in the given line for spacing issues.
+        Yields a Result for each issue found.
+
+        :param file:
+            The content of the file currently being inspected.
+        :param filename:
+            The name of the file currently being inspected.
+        :param line:
+            The content of the line currently being inspected.
+        :param line_number:
+            The current line number.
+        :param statement_spacing:
+            The number of spaces required on each side of a statement tag.
+        """
+        for m in self.STATEMENT_REGEX.finditer(line):
+            match = m.group('content')
+            if not has_required_spacing(match, statement_spacing):
+                diff = generate_spacing_diff(
+                    file, filename, line, line_number, m,
+                    statement_spacing)
+                yield Result.from_values(
+                    origin=self,
+                    message='Statement blocks should be spaced with '
+                            '`{0}` spaces on each side.'.format(
```

Comment:
unnecessary `0` inside format placeholder :)

Reply:
I would rephrase the message a bit:
```
Statement blocks not spaces with {} spaces on each side.
```

Reply:
So it gets a more passive voice. Using `should` could sound like some recommendation from our side, although the coala-user was the one specifying the option ;)